duckbot/logging/exception_logs.py

```
@@ -0,0 +1,18 @@
+import logging
+import sys
+import traceback
+
+from discord.ext import commands
+
+
+class ExceptionLogs(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.Cog.listener("on_command_error")
+    async def log_exceptions(self, context, exception):
+        logger = logging.getLogger("discord")
+        exception_string = "".join(traceback.format_exception(etype=type(exception), value=exception, tb=exception.__traceback__))
+        logger.error(exception_string)
+        print("Brother, ignoring exception in command {}:".format(context.command), file=sys.stderr)
```

Prefer f-strings to explicit `format` calls. This can be `f"Brother, ignoring exception in command {context.command}:"`

---

Agreed. Changed.